### PR TITLE
Fix request flow

### PR DESCRIPTION
**Pull request by relaido**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Fix to always request the VAST API whether case Android and IOS.
Also, fix to save uuid to cookie instead of local storage.

### CODE REVIEW

File: test/spec/modules/relaidoBidAdapter_spec.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const setUAMobile = () => { window.navigator.__defineGetter__('userAgent', funct
 
 describe('RelaidoAdapter', function () {
   const relaido_uuid = 'hogehoge';
+  window.document.cookie = `${UUID_KEY}=${relaido_uuid}`
```
Comment (robertrmartinez) [inline test/spec/modules/relaidoBidAdapter_spec.js:14]: please do not use direct cookie setting in your tests.

Unit tests are run across all browsers and this causes flaky results at times.

Instead, you should stub the utils.getCookie and utils.setCookie functions to perform whatever you want.

**Review by robertrmartinez — CHANGES_REQUESTED** [review]
looks fine,

But please update the tests to stub the storage functions instead of directly setting on window.

File: test/spec/modules/relaidoBidAdapter_spec.js (hunk 1 above)
Comment (relaido) [reply test/spec/modules/relaidoBidAdapter_spec.js:14]: Thank you for the review.

For various reasons, close this pull request

I will send you a new pull request that includes this one immediately.

I'll fix the test code at that time.

**Comment by relaido** [comment]
Due to some content that arose that we wanted to add to the site as soon as possible

**Pull request closed without merging**